orchagent/neighorch.cpp

```diff
@@ -30,6 +30,13 @@ bool NeighOrch::addNextHop(IpAddress ipAddress, string alias)
 {
     SWSS_LOG_ENTER();
 
+    if (hasNextHop(ipAddress))
```

Comment:
Do you want to take care of out-of-sequence in the delete case as well?

Reply:
any update on this @stcheng ?